pkg/util/metadata/metadata.go

```diff
@@ -205,3 +209,21 @@ func Get(order string) (*Metadata, error) {
 	}
 	return metadataCache, nil
 }
+
+// GetInstanceID from metadata service
```

Comment:
needs to be updated, as now it retrieves from both metadata servie and config drive

Reply:
ok